crates/rethnet_eth/src/remote.rs

```diff
@@ -69,6 +69,18 @@ where
     s.serialize_str(&result)
 }
 
+/// for use with serde's deserialize_with on fields of hexadecimal strings that should be
+/// parsed as Option<u64>
+pub fn optional_u64_from_hex<'de, D>(deserializer: D) -> Result<Option<u64>, D::Error>
```

Comment:
Is it common to write code under the `src` directory that's only used in test code?

I'm pretty sure this isn't a big deal because this will not be part of the binary (unlike in javascript, where the code will probably end up in the user's `node_modules` if you write it under `src`). Just curious if it's something that you guys are already doing elsewhere.

Reply:
I realized after writing this that you just moved an already existing function. I'm still curious about this, but there's no need to do anything about it.

Reply:
It's a good question, and I'm not sure what the answer is.

Reply:
Often you'll create a separate module (that is never published to crates.io) which contains test helpers. You can include that as a dev-dependency in your Cargo.toml